src/components/hierarchicalfacets/precompile-templates/subHierarchicalFacetsTemplate.handlebars

```diff
@@ -1,16 +1,22 @@
-<li>
+<li role="treeitem" aria-level="{{level}}" aria-expanded="{{#unless (validateOpenState value)}}true{{else}}false{{/unless}}" {{#if children}}aria-owns="children-{{value}}"{{/if}}>
```

Comment:
The implementation uses role="tree" but mixes it with checkboxes, which violates ARIA patterns.  According to https://www.w3.org/WAI/ARIA/apg/patterns/treeview/, trees are for hierarchical navigation, not selection.

Option A: Use role="group" instead of "tree"
Option B: If tree navigation is desired, remove checkboxes and use aria-selected

Reply:
Key changes:
- remove role="treeitem", role="tree"
- use role="group" with aria-labelledby
- move aria-expanded to "button" (now it's in `li`)